### DISCUSSION TITLE
`follow_current_file = { enable = true }` Not Functioning Properly - Any Solutions?

### DISCUSSION THREAD

**Discussion by LoneExile** [nvim-neo-tree/neo-tree.nvim#1041] [Q&A] [answered]
I'm using `<cmd>Neotree action=focus source=filesystem position=float toggle=true<CR>`, 

However, it doesn't seem to follow the current file I'm editing when I try to use `<C-o>` and `<C-i>`.

```lua
filesystem = {
-- ...
  follow_current_file = { enable = true },
-- ...
},
```

**Comment by cseickel** [comment, marked as answer]
A float can't follow the current file, because a float will close whenever you focus another window. It doesn't make any sense.

**Comment by pablomarelli** [comment]
For someone looking for a solution on how to make the float follow the file you're on, you have to set the keymap to reveal as explained here:

https://github.com/LazyVim/LazyVim/discussions/1355#discussion-5537146

**Reply by TrevorW-code** [reply]
If your neotree plugin is configured like this:

```
.
├── init.lua
├── lazy-lock.json
└── lua
    ├── config
    │   └── lazy.lua
    └── plugins
        ├── autocomplete.lua
        ├── lsp.lua
         ...
        └── neotree.lua
```

You can also add a remap in your `init.lua`. Here I've used `<leader>pv`

```
vim.keymap.set("n", "<leader>pv", "<cmd>Neotree toggle reveal<CR>", { desc = "Toggle Neo-tree & Reveal File" })
```

This should both toggle neotree & reveal the file (when toggled on) with one command.
